res/controllers/common-controller-scripts.js

```diff
@@ -32,7 +32,7 @@ String.prototype.toInt = function() {
 
 // Causes script print() calls to appear in the log file as well
 var print = function(string) {
-    engine.log(string);
+    console.log(string);
```

Comment:
same unnecessary mass-find-and-replace as #4519

please refrain from them in the future.

Reply:
This is not unnecessary. Every script would print a bunch of deprecation warnings without this.

Reply:
Ah, I wouldn't have put that deprecation warning in there and just documented is as deprecated.